src/common/reconciler.go

```diff
@@ -361,3 +362,34 @@ func (r *ServiceReconciler) handleRepositories(service *opslevel.Service, regist
 		}
 	}
 }
+
+func (r *ServiceReconciler) handleProperties(service *opslevel.Service, registration opslevel_jq_parser.ServiceRegistration) {
+	for def, val := range registration.Properties {
+		definition := opslevel.NewIdentifier(def)
+		owner := opslevel.NewIdentifier(string(service.Id))
+		// TODO: hack to get around NewJSONInput not reading strings properly.
```

Comment:
We need a separate function for parsing JSON from strings OR we need to add logic to `opslevel.NewJSONInput` to handle strings this way. Without this, strings are sent unquoted and the server outputs `unexpected token at`...

Reply:
see https://github.com/OpsLevel/opslevel-go/pull/365